src/containers/Transactions/index.js

```diff
@@ -66,22 +64,10 @@ const Transaction = (props) => {
 
   function renderSummary() {
     const type = data.raw.tx.TransactionType
-    const status =
-      data.raw.meta.TransactionResult === SUCCESSFULL_TRANSACTION ? (
-        <div className="status success">
-          <img src={successIcon} alt="success" />
-          {t('success')}
-        </div>
-      ) : (
-        <div className="status fail">
-          <img src={infoIcon} alt="fail" />
```

Comment:
So is the `infoIcon` no longer used?

Reply:
Correct.  I will clean it up.  Now it will show the whole error code with a link.